kitchen-tests/kitchen.exec.macos.yml

```
@@ -27,6 +27,7 @@ lifecycle:
 
 platforms:
   - name: macos-latest # arm64
```

for clarity and future proofing when they take 15 out of beta and make it the latest.
```suggestion
  - name: macos-14 # arm64
```